### DISCUSSION TITLE
subprocess.CalledProcessError: Command '['/home/u......] died with <Signals.SIGKILL: 9>.

### DISCUSSION THREAD

**Discussion by Sreekanth-DSAI** [huggingface/autotrain-advanced#498] [General]
Hi Abhishek, I am getting this error "  File "/home/ubuntu/llama_train/venv/lib/python3.10/site-packages/accelerate/commands/launch.py", line 643, in simple_launcher
    raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)
subprocess.CalledProcessError: Command '['/home/ubuntu/llama_train/venv/bin/python3', '-m', 'autotrain.trainers.clm', '--training_config', 'my_project/training_params.json']' died with <Signals.SIGKILL: 9>.
". How to give validation dataset here. it is saying valid data: None

**Comment by abhishekkrthakur** [comment]
llm finetuning ignores validation data.
sigkill is due to out of memory for either cpu or gpu
